ci/cloudbuild.yaml

```diff
@@ -13,6 +13,10 @@ steps:
   id: 'Create version tags'
   args: ['python3', 'ci/create-version-tags.py']
 
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: '/bin/bash'
+  args: ['-c', 'docker pull gcr.io/$PROJECT_ID/ts-bridge:latest || exit 0']
```

Comment:
This would hide any actual pull errors though. What do you want to do here? I may be able to offer something more selective here.